ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron/Mempool.hs

```diff
@@ -106,12 +106,6 @@ instance ApplyTx ByronBlock where
     where
       validationMode = CC.ValidationMode CC.NoBlockValidation Utxo.TxValidationNoCrypto
 
-  reapplyTxSameState cfg tx st =
-      validationErrorImpossible $
-        applyByronGenTx validationMode cfg tx st
-    where
-      validationMode = CC.ValidationMode CC.NoBlockValidation Utxo.NoTxValidation
```

Comment:
Does this mean that we no longer need the `NoTxValidation` mode in `cardano-ledger`?

Reply:
Indeed not. @edsko and I have been looking for a ticket about this (that we could close), but we couldn't find it. Do you know which one that would be?

Reply:
I'm not sure that there has been a ticket in ledger that specifically speaks to removing `NoTxValidation`.